src/components/Form.vue

```diff
@@ -1,15 +1,92 @@
 <template>
- <div>
-    <b-field label="Enter Todo Here..">
-          <b-input v-model="message"  @keypress.native.enter="addTodo"></b-input>
-      </b-field>
-</div>
+  <div>
+    <form @submit.prevent="onSubmit">
+      <section v-if="seen">
+        <b-field label="Name" :type="errors.has('name') ? 'is-danger': ''" :message="errors.has('name') ? errors.first('name') : ''">
+          <b-input type="text" placeholder="Name" name="name" v-model="name" v-validate="'required|alpha'" :class="{'input': true, 'is-danger': errors.has('name') }"></b-input>
+        </b-field>
+
+        <b-field label="Email" :type="errors.has('email') ? 'is-danger': ''" :message="errors.has('email') ? errors.first('email') : ''">
+          <b-input type="text" placeholder="Email" name="email" v-model="email" v-validate="'required|email'" :class="{'input': true, 'is-danger': errors.has('email') }"></b-input>
+        </b-field>
+  
+        <b-field label="Password" :type="errors.has('password') ? 'is-danger': ''" :message="errors.has('password') ? errors.first('password') : ''">
+          <b-input type="password" name="password" placeholder="Password" v-model="password" v-validate="'required|max:16|min:8'" :class="{'input': true, 'is-danger': errors.has('password') }"></b-input>
+        </b-field>
+  
+        <b-field label="Confirm Password" :type="errors.has('conf-password') ? 'is-danger': ''" :message="errors.has('conf-password') ? errors.first('conf-password') : ''">
+          <b-input type="password" name="conf-password" v-model="confirm" placeholder="Confirm Password" v-validate="'required|max:16|min:8'" :class="{'input': true, 'is-danger': errors.has('conf-password') }"></b-input>
+        </b-field>
+        
+        <b-field label="Task" :type="errors.has('message-task') ? 'is-danger': ''" :message="errors.has('message-task') ? errors.first('message-task') : ''">
+          <b-input v-model="message" name="message-task" v-validate="'required'" placeholder="Enter Your Todo's here!" :class="{'input': true, 'is-danger': errors.has('conf-password') }"></b-input>
+        </b-field>
+  
+        <div class="two-button">
+          <button type="button" class="button is-warning" value="cancel">Cancel</button>
+          <button type="submit" class="button is-success" value="submit" :disabled="errors.any()" v-on:click="submit">Submit</button>
+        </div>
+      </section>
+      
+    </form>
+    <b-notification type="is-success" has-icon v-if="isActive">
+      Form is successfully submitted ! :)
+      <br/> Go to the list
+    </b-notification>
+  </div>
 </template>
 
 <script>
+let count = 0;
```

Comment:
what is this doing here??

Reply:
sir this count variable is assigning the value to id.